sys-fs/btdu/btdu-0.5.1.ebuild

```diff
@@ -0,0 +1,39 @@
+# Copyright 2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+DUB_DEPENDENCIES=(
+	"ae@0.0.3236"
+	"btdu@0.5.1"
+	"btrfs@0.0.18"
+	"emsi_containers@0.9.0"
+	"ncurses@1.0.0"
+)
+# gdc-13 is disabled due to: https://gcc.gnu.org/bugzilla/show_bug.cgi?id=111650
+# gdc-14 is disabled due to: https://gcc.gnu.org/bugzilla/show_bug.cgi?id=116373
+DLANG_COMPAT=( dmd-2_10{6..9} ldc2-1_3{5..9} )
+inherit dlang-single dub
+
+DESCRIPTION="Sampling disk usage profiler for btrfs "
+HOMEPAGE="https://github.com/CyberShadow/btdu"
+SRC_URI="${DUB_DEPENDENCIES_URIS}"
+
+LICENSE="GPL-2"
+LICENSE+=" Boost-1.0 GPL-2 MIT MPL-2.0"
+SLOT="0"
+KEYWORDS="~amd64"
+RESTRICT="test"
+
+RDEPEND="${DLANG_DEPS}"
```

Comment:
The eclass should be setting this for us by default, right? (These variables accumulate when set by eclasses, see PMS)

Reply:
Yeah, I think that's a better design. I guess I need to check if `dlang-single` or `dlang-r1` has been inherited.